pkg/resourcegenerator/resourceutils/metadata_test.go

```diff
@@ -31,7 +31,7 @@ func TestSetResourceLabels(t *testing.T) {
 	}
 
 	SetApplicationLabels(sa, app)
-	assert.True(t, len(sa.GetLabels()) == 6)
+	assert.True(t, len(sa.GetLabels()) == 8)
```

Comment:
It would create a better test if you tested against an `expectedLabels` map with `assertEqual(sa.GetLabels, expectedLabels)` for example